### PR TITLE
Added GATK EstimateLibraryComplexity wrapper

### PR DESCRIPTION
**Pull request by fgvieira**

### CODE REVIEW

File: bio/gatk/estimatelibrarycomplexity/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  Run gatk EstimateLibraryComplexity
+authors:
+  - Filipe G. Vieira
+input:
+  - SAM/BAM/CRAM file
+output:
+  - metrics file
+notes: |
+  * The `java_opts` param allows for additional arguments to be passed to the java compiler, e.g. "-XX:ParallelGCThreads=10" (not for `-XmX` or `-Djava.io.tmpdir`, since they are handled automatically).
+  * The `extra` param alllows for additional program arguments.
```
Comment (jafors) [inline bio/gatk/estimatelibrarycomplexity/meta.yaml:12]:
```suggestion
  * The `extra` param allows for additional program arguments.
```

File: bio/gatk/estimatelibrarycomplexity/test/Snakefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        bam = "{sample}.bam"
+    output:
+        metrics = "{sample}.metrics"
+    log:
+        "logs/{sample}.log"
+    params:
+        extra = "",
+	java_opts="", # optional
+    resources:
+        mem_mb = 1024,
```
Comment (jafors) [inline bio/gatk/estimatelibrarycomplexity/test/Snakefile:12]:
```suggestion
        bam="{sample}.bam"
    output:
        metrics="{sample}.metrics"
    log:
        "logs/{sample}.log"
    params:
        extra="",
	java_opts="", # optional
    resources:
        mem_mb=1024,
```

**Review by jafors — CHANGES_REQUESTED** [review]
Thanks for this addition, the suggestions are purely cosmetic.

**Comment by fgvieira** [comment]
Always forget about the spaces... sorry about that! :smile:

**Review by jafors — APPROVED** [review]

**Pull request merged**